x/nft/codec.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package nft;
+
+message NonFungibleToken {
+    bytes id = 1;
+    bytes owner = 2;
+    repeated ApprovalSet approvalSet = 3;
+}
+
+message ApprovalSet {
+    string action = 1;
+    repeated Approval approvals = 2;
+}
+
+message Approval {
+    bytes toAccount = 1;
```

Comment:
I think this has to be changed to `address` or something, a bit cleaner naming. As there is only one account involved - no need for any additional explanation

Reply:
yes, it should be address kind. good point.